includes/class-event.php

```diff
@@ -132,6 +135,7 @@ public function save() {
 
 		if ( $this->exists() ) {
 			$success = Events_Store::_update_event( $this->id, $row_data );
+			// TODO: update last modified
```

Comment:
Are you going to implement these TODOs?

Reply:
Was thinking to address is separately. There's not really any harm in them "being stale" atm, but if ever exposing them w/ their own getters or logic, then I'd want the class to keep them fresh.

Probably means setting the dates within save() and passing it to the store methods, rather than setting them within the store.

Reply:
Okay turns out this was much easier than I thought since everything is funneled through the same flow now!

Both TODOs account for :)